templates/openstack/what-is-openstack.html

```diff
@@ -0,0 +1,115 @@
+{% extends "openstack/_base_openstack.html" %}
+
+{% block title %}What is Openstack | OpenStack{% endblock %}
+{% block meta_description %}OpenStack is an open source cloud computing platform that allows businesses to control large pools of compute, storage and networking in a data centre.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>What is OpenStack?</h1>
+      <p>OpenStack is an open source cloud computing platform that allows businesses to control large pools of compute, storage and networking in a data centre.</p>
+      <p>The fact that OpenStack is open source means that anyone that chooses to use it, can access the source code, make changes, and share these with the community. One of the key benefits of this model is that the source code can then be checked by a much larger set of people than proprietary code, which is restricted to its owners.</p>
+      <p>Governed by the OpenStack Foundation, there are more than 34,000 individual contributors and over 550 companies that participate in the project. </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+    <div class="row">
+      <div class="col-8">
+        <h2>Companies involved in OpenStack</h2>
+        <p>OpenStack was originally launched in July 2010 by Rackspace and NASA as an open source initiative that combined NASA’s Nebula platform and Rackspace’s Cloud Files platform.</p>
+        <p>Today, hundreds of companies contribute to OpenStack code, with many more playing a part in the broader community.</p>
+      </div>
+    </div>
+</section>
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="col-12"> 
+    <ul class="row"> 
+      <h3 class="p-muted-heading">Influential contributors to openStack</h3>
```

Comment:
The h3 tag fails validation. Apparently it is not allowed a child UL. You might need to move it up.